.changeset/gold-buttons-call.md

```diff
@@ -0,0 +1,7 @@
+---
+"@sap-ux/cds-annotation-parser": patch
+"@sap-ux/cds-odata-annotation-converter": patch
+"@sap-ux/i18n": patch
```

Comment:
You also need to add the new package here for it to be published after this PR is merged.

Reply:
updated the changeset, with minor.